### ISSUE TITLE
Pre-trained models take 2 - tracker

### ISSUE THREAD

**Issue by SeanNaren** [SeanNaren/deepspeech.pytorch#199]
On each of the datasets provided, we must train a Deepspeech model and document exactly how to reproduce these results! The overall architecture is encompassed in this command:

```
python train.py  --rnn_type gru --hidden_size 800 --hidden_layers 5 --checkpoint --visdom --train_manifest /path/to/train_manifest.csv --val_manifest /path/to/val_manifest.csv --epochs 100 --num_workers $(nproc) --cuda
```

In the above command you must replace the manifests paths with the correct paths to the dataset. A few notes:

- Train till convergence (should get a nice smooth training curve hopefully!)
- For smaller datasets, you may need to reduce the learning rate annealing by adding the flag `--learning anneal` and setting it to a smaller value, like `1.01`. For larger datasets, the default is fine (up to around 4.5k hours from internal testing on the deepspeech.torch version)

If you use augmentation or noise injection give as much details as possible for me to wrap up it all nicely.

A release will be cut from the DeepSpeech package that will have the models, and a reference to the latest release added to the README to find latest models!

Progress tracker for datasets:

- [x] AN4
- [x] TEDLium
- [x] LibriSpeech

Let me know if you plan on working on running any of these, and I'll update the ticket with details!

**Comment by SeanNaren** [comment]
This ties into #193

**Comment by ryanleary** [comment]
Also consider training unidirectional models.

**Comment by SeanNaren** [comment]
@ryanleary I think for the first release I suggest giving bi-directional models, and in a separate ticket worry about uni-directional, just to fix the reproducibility issues we currently have.

Librispeech is completed, information of the trained model below:

Commit hash: [87fd100fe455c4063bf3a8ed8df4f95b31ef6e00](https://github.com/SeanNaren/deepspeech.pytorch/tree/87fd100fe455c4063bf3a8ed8df4f95b31ef6e00)

Training command: 
```
python train.py  --rnn_type gru --hidden_size 800 --hidden_layers 5 --checkpoint --visdom --train_manifest data/libri_train_manifest.csv --val_manifest data/libri_val_manifest.csv --epochs 15 --num_workers $(nproc) --cuda --checkpoint --batch_size 10 --learning_anneal 1.1
```
|Dataset          | WER      | CER      |
|-----------------|:--------:|:--------:|
|Librispeech clean | 11.27     | 3.09      |
|Librispeech other | 30.74      | 10.97      |

**Comment by SeanNaren** [comment]
Training the TEDlium model currently. I do not have the compute resources to sufficiently train the combined model however.

I'm also up for discussion around whether we should include voxforge. Voxforge is sometimes hacked around to turn into a testing set (in some baidu papers I've seen this for specific accents). @ryanleary any thoughts on that?

**Comment by SeanNaren** [comment]
TedLIUM is completed, information of the trained model below:

Commit hash: [87fd100fe455c4063bf3a8ed8df4f95b31ef6e00](https://github.com/SeanNaren/deepspeech.pytorch/tree/87fd100fe455c4063bf3a8ed8df4f95b31ef6e00)

Training command: 
```
python train.py  --rnn_type gru --hidden_size 800 --hidden_layers 5 --checkpoint --visdom --train_manifest data/ted_train_manifest.csv --val_manifest data/ted_val_manifest.csv --epochs 15 --num_workers $(nproc) --cuda --checkpoint --batch_size 10 --learning_anneal 1.1
```
|Dataset          | WER      | CER      |
|-----------------|:--------:|:--------:|
|Ted test | 31.04     | 10.00     |

**Comment by SeanNaren** [comment]
For the sake of getting the academic models out fast I'll be making a separate ticket for the combined model.

**Comment by SeanNaren** [comment]
Done! release [here[(https://github.com/SeanNaren/deepspeech.pytorch/releases/tag/v1.1).

**Comment by yyang29** [comment]
Hi guys, just to confirm, is there any unidirectional model released yet?

**Comment by oplatek** [comment]
Hi guys, as above is there any unidirectional model? If yes is there a streaming interface for it?

Hints appreciated in what needs to be done to have both a) the model b) the streaming decoding interface

**Comment by heibaidaolx123** [comment]
Hi guys, any update?
